minimumCostOfBuyingCandies.js

```diff
@@ -0,0 +1,20 @@
+// This file contains solution to the following problem:
+// https://leetcode.com/problems/minimum-cost-of-buying-candies-with-discount/
+
+const minimumCost = (cost) => {
+  const sortedPrices = cost.sort((a, b) => b - a);
+  const freeCandy = 3;
+  let result = 0;
+  let counter = 0;
+  let temp = 0;
+  while (counter < sortedPrices.length) {
```

Comment:
I feel like it'd be simpler to write this `while` loop as a `for` loop in this particular case, lets do it just to practice if nothing else.